templates/default/nginx.upstart.erb

```diff
@@ -0,0 +1,47 @@
+# taken from opscode-cookbook/nginx
+# https://raw.github.com/opscode-cookbooks/nginx/master/templates/default/nginx-upstart.conf.erb
+
+description "nginx http daemon"
+
+start on (local-filesystems and net-device-up IFACE=lo)
+stop on runlevel [!<%= node['openresty']['service']['upstart']['runlevels'] %>]
+
+env DAEMON=<%= node['openresty']['binary'] %>
+env PID=<%= node['openresty']['pid'] %>
+env CONFIG=<%= node['openresty']['source']['conf_path'] %>
+
+# set limit for upstart jobs
+limit nofile 1048576 1048576
+respawn
+<% if node['openresty']['service']['upstart']['respawn_limit'] %>
+respawn limit <%= node['openresty']['service']['upstart']['respawn_limit'] %>
+<% end %>
+
+pre-start script
+  ${DAEMON} -t
+  if [ $? -ne 0 ]; then
+    exit $?
+  fi
+  echo "[`date -u +%Y-%m-%dT%T.%3NZ`] (sys) Starting openresty" >> /var/log/openresty.log
+end script
+
+pre-stop script
+  echo "[`date -u +%Y-%m-%dT%T.%3NZ`] (sys) Stopping openresty" >> /var/log/openresty.log
```

Comment:
This log diverges from actually keeping everything in /var/log/nginx. Can you please refactor this to actually use upstart's logging capabilities?